base/linalg/symmetric.jl

```diff
@@ -473,7 +473,7 @@ for (funm, func) in ([:logm,:log], [:sqrtm,:sqrt])
     @eval begin
         function ($funm){T<:Real}(A::Symmetric{T})
             F = eigfact(A)
-            if isposdef(F)
+            if all(λ -> λ ≥ 0, F.values)
```

Comment:
Why this change? `isposdef(F)` is more clear IMO and [does the same thing](https://github.com/JuliaLang/julia/blob/7eefa45bdedd81da4742c5a1649b241c09fd6424/base/linalg/eigen.jl#L30)? Same just below too.